Source/WebCore/page/cocoa/MemoryReleaseCocoa.mm

```diff
@@ -47,8 +47,6 @@
 void platformReleaseMemory(Critical)
 {
 #if PLATFORM(IOS_FAMILY) && !PLATFORM(IOS_FAMILY_SIMULATOR) && !PLATFORM(MACCATALYST)
-    // FIXME: Remove this call to GSFontInitialize() once <rdar://problem/32886715> is fixed.
-    GSFontInitialize();
```

Comment:
since this is the only call, we should probably revert the whole commit: https://trac.webkit.org/changeset/218643/webkit

Reply:
Done!